problemtools/verifyproblem.py

```diff
@@ -703,7 +703,7 @@ def _build_junk_modifier(desc, pattern, repl):
 
 _JUNK_MODIFICATIONS = [
     _build_junk_modifier('spaces added where there already is whitespace', r'\s', lambda m: m.group(0) + ' ' * random.randint(0, 5)),
```

Comment:
Should this one also be updated (to `randint(1, 5)`)?

Reply:
Yes, it should. Thanks. Will fix.